src/main/resources/META-INF/fieldtomethodtransformers.js

```diff
@@ -19,7 +19,7 @@ function initializeCoreMod() {
             },
             'transformer': function(classNode) {
                 var asmapi=Java.type('net.minecraftforge.coremod.api.ASMAPI')
-                var fn = asmapi.mapField('field_204517_c') // potion field - remap to mcp if necessary
+                var fn = asmapi.mapField('field_204517_c') // fluid field - remap to mcp if necessary
                 asmapi.redirectFieldToMethod(classNode, fn, asmapi.mapMethod('getFluid'))
```

Comment:
if getFluid is a method that has no srg name (i.e. added by a forge patch),  you can just call `asmapi.redirectFieldToMethod(classNode, fn, 'getFluid')`, as it does not need to be remapped

Reply:
Oops, completely missed that lol.
/me hides in shame
Will fix when I'm back home later